interface/src/state/mint.rs

```diff
@@ -63,6 +69,8 @@ impl Mint {
     }
 
     #[inline(always)]
+    // JC: same as with the other COption<Pubkey> fields, check that this is the
+    // current behavior
```

Comment:
Same as the current behaviour: https://github.com/solana-program/token/blob/main/program/src/state.rs#L80